src/macros.jl

```diff
@@ -60,8 +60,13 @@ macro polyvariable(args...)
   end
 
   # separate out keyword arguments
-  kwargs = filter(ex->isexpr(ex,:kw), extra)
-  extra = filter(ex->!isexpr(ex,:kw), extra)
+  if VERSION < v"0.6-"
```

Comment:
The version cutoff is `v"0.6.0-dev.1934"` for this one, it is used in https://github.com/JuliaOpt/JuMP.jl/commit/b8a321e2862179a00087e65ab80414b49e79d714 :-P